bundles/org.openhab.binding.argoclima/README.md

```diff
@@ -0,0 +1,429 @@
+
+  The Argo protocol is fully supported by the [IRremoteESP8266](https://github.com/crankyoldgit/IRremoteESP8266) library, and [Tasmota](https://github.com/arendst/Tasmota). Hence, any [Tasmota-IR](https://tasmota.github.io/docs/Tasmota-IR/)-compatible device would be able to send iFeel data to the HVAC (and the device itself can be controlled via openHAB's [MQTT Binding](https://www.openhab.org/addons/bindings/mqtt/)).
+
+  For example, (by default) the following command sent to ```cmnd/<tasmota_dev_id>/irhvac``` channel does send the iFeel temperature.
+
+  ```jsonc
+  // Hint: You may use JINJA transform to fill the room temperature (in Celsius) from a channel.
+  //       The status of IR command execution is available through [stat/<tasmota_dev_id>/RESULT] channel
+  { "Vendor": "Argo", "Model": "WREM3", "Command": "iFeel Report", "SensorTemp": <the_room_temperature_value> }
+  ```
+
+## Argo Web APP details
+
+The supported devices will typically have a web interface similar to the one shown on the image below.
+![Argo Web APP](doc/ArgoWebAPP_UI.png)
+
+For example, the original web application for **Ulisse 13 DCI ECO** air conditioner is available through [http://31.14.128.210/UI/WEBAPP/webapp.php?logo=Argo](http://31.14.128.210/UI/WEBAPP/webapp.php?logo=Argo) link, as referenced in the [manual](https://www.pumppujapaneli.fi/flaria/media/ulisse-kayttoohje.pdf).
+
+## Credits
+
+The author would like to thank the following individuals:
+
+- Maintainers of [IRremoteESP8266](https://github.com/crankyoldgit/IRremoteESP8266) and [Tasmota](https://github.com/arendst/Tasmota) projects - for reviewing an accepting the infrared-related part of Argo protocol into their libraries!
+- [@nyffchanium](https://github.com/nyffchanium) for creating an awesome [Argoclima integration for HomeAssistant](https://github.com/nyffchanium/argoclima-integration) which was used to confirm & speed up the analysis of device's protocol.
+While most of the learnings come from analyzing the JavaScript code in Argo's own application and network captures, the HA integration has proven **invaluable** as a secondary/confirmed source and allowed to validate a few concepts early on!
+  - In case you're experiencing issues, make sure to read the HomeAssistant binding's [README](https://github.com/nyffchanium/argoclima-integration/blob/master/readme.md) as well, for useful troubleshooting info!
+- [@lallinger](https://github.com/lallinger) for a [dummy server](https://github.com/nyffchanium/argoclima-integration/tree/master/dummy-server) which was the idea that served as the cornerstone of the stub server built into this binding (which later got extended to do something useful, instead of just keeping the device happy).
+
+## Disclaimer
+
+This project is not affiliated with, funded, or in any way associated with Argoclima S.p.A.
+All third-party product, company names, logos and trademarks™ or registered® trademarks remain the property of their respective holders.
```

Comment:
@kaikreuzer - sorry for tagging you directly (well, half-sorry, since I'm still doing it 😄) - do we have some project-wise disclaimer like this? It feels a bit inconsistent to have this at binding level, since I guess most bindings do not have this. On the other hand, it must be on binding-level to mention specific companies we are not affiliated with. From a legal perspective, is this needed at all?

Reply:
@lsiepel, @mbronk - FTR, my comment here is generic and doesn't have to be considered for the PR at hand. Depending on response from @kaikreuzer we can adjust (if needed) even after this PR is merged.